Mathlib/MeasureTheory/Integral/Lebesgue.lean

```diff
@@ -800,6 +800,14 @@ theorem lintegral_indicator_one {s : Set α} (hs : MeasurableSet s) :
     ∫⁻ a, s.indicator 1 a ∂μ = μ s :=
   (lintegral_indicator_const hs _).trans <| one_mul _
 
+theorem measure_eq_measure_iff_lintegral_eq_lintegral (ν : Measure α) :
+    μ = ν ↔ ∀ f : α → ℝ≥0∞, Measurable f → ∫⁻ a, f a ∂μ = ∫⁻ a, f a ∂ν := by
+  refine ⟨fun h _ _ ↦ by rw[h], ?_⟩
+  intro h
+  ext s hs
+  simp[← lintegral_indicator_one hs, h]
```

Comment:
That simp does not close a goal: it should be replaced by a `simp only` (use `simp?` to get a suggestion for the replacement). Also put a space between `simp` and `[`.